step2-07/demo/src/components/TodoListItem.tsx

```diff
@@ -27,13 +27,11 @@ class TodoListItem extends React.Component<TodoListItemProps, {}> {
   }
 }
 
-function mapStateToProps({ todos }: Store) {
-  return {
-    todos
-  };
+function mapStateToProps({ todos }: Store): Pick<TodoListItemProps, 'todos'> {
```

Comment:
For some reason I had to use Pick on these ones to avoid a compiler error (if I used Partial here, TodoList was giving an error that `id` wasn't a valid property)

Reply:
If you want to use mapStateToProps, mapDispatchToProps, please, consider already defined types from react-redux `MapStateToProps`, `MapDispatchToProps`. This way types will be more strict and easier to use